### PR TITLE
docs(samples): updated NUMERIC samples

### PR DESCRIPTION
**Pull request by skuruppu**
Added specific samples for NUMERIC instead of modifying the existing datatypes samples.

### CODE REVIEW

File: samples/datatypes.js [hunk 1]
```diff
@@ -611,6 +607,97 @@ async function queryWithTimestamp(instanceId, databaseId, projectId) {
...
+  /**
+   * TODO(developer): Uncomment the following lines before running the sample.
+   */
+  // const projectId = 'my-project-id';
+  // const instanceId = 'my-instance';
+  // const databaseId = 'my-database';
+
+  // Creates a client
+  const spanner = new Spanner({
+    projectId: projectId,
```
Comment (JustinBeckwith) [inline samples/datatypes.js:624]: So typically, we don't bother passing `projectId` in like this, and rather rely on the library figuring it out on it's own.  There are places we need it, but I kind of think we should keep the constructor parameterless where we can.

File: samples/datatypes.js [hunk 2]
```diff
@@ -611,6 +607,97 @@ async function queryWithTimestamp(instanceId, databaseId, projectId) {
   // [END spanner_query_with_timestamp_parameter]
 }
 
+async function addNumericColumn(instanceId, databaseId, projectId) {
```
Comment (JustinBeckwith) [inline samples/datatypes.js:610]: For all new samples, we're really trying to keep to one snippet per file:
https://g3doc.corp.google.com/company/teams/cloud-devrel/dpe/samples/nodejs.md?cl=head

File: samples/system-test/spanner.test.js [hunk 3]
```diff
@@ -774,16 +774,38 @@ describe('Spanner', () => {
...
-  // query_with_numeric_parameter
-  it('should use a NUMERIC query parameter to query record from the Venues example table', async () => {
+  // add_numeric_column
+  it('should add a Revenue column to Venues example table', async () => {
     const output = execSync(
-      `${datatypesCmd} queryWithNumeric ${INSTANCE_ID} ${DATABASE_ID} ${PROJECT_ID}`
+      `${datatypesCmd} addNumericColumn "${INSTANCE_ID}" "${DATABASE_ID}" ${PROJECT_ID}`
+    );
+
+    assert.match(
```
Comment (JustinBeckwith) [inline samples/system-test/spanner.test.js:783]: If you don't need a regex here, you can use `assert.include`:
```js
assert.include(output, `Waiting for ...`);
```

**Review by JustinBeckwith — COMMENTED** [review]
The feedback here is mostly `nit` in nature, and maybe stuff we want to try applying more broadly, and not just in this PR.  

Other issue I see here - when you use the `fix:` conventional commit prefix, that signals that this change should trigger a release of the library.  But really - there are no material changes to the shipped code, just the samples.  For that, I would suggest using a `docs(samples): ` prefix :)

**Review by olavloite — APPROVED** [review]
From a purely Spanner - Numeric sample point of view this LGTM (but note the comments from @JustinBeckwith )

File: samples/datatypes.js (hunk 2 above)
Comment (skuruppu) [reply samples/datatypes.js:610]: Ah yes, that makes sense. I've now moved the new samples into their own files.

File: samples/datatypes.js (hunk 1 above)
Comment (skuruppu) [reply samples/datatypes.js:624]: Oh interesting. Yeah I suppose we have the GCLOUD_PROJECT env var that could be read. I understood this as, customers may not know how to set that (or maybe they're not comfortable using env vars) so we explicitly set them.

But if it is against the style guide, we can do a repo-wide cleanup of this.

I'll leave it as it is for now, just for consistency with how existing samples are set up.

File: samples/system-test/spanner.test.js (hunk 3 above)
Comment (skuruppu) [reply samples/system-test/spanner.test.js:783]: Thanks, that's much neater.

**Comment by skuruppu** [comment]
Thanks so much for the review and feedback @olavloite and @JustinBeckwith.

**Review by JustinBeckwith — APPROVED** [review]

**Pull request merged**